bin/git-authors

```diff
@@ -43,12 +23,47 @@ authors() {
   fi
 }
 
-#
-# authors.
-#
+if [[ -z "$file" ]]; then
+  file=$(find . -mindepth 1 -maxdepth 1 \( -iname '*authors*' -o -iname '*contributors*' \) | head -n1)
+  if [[ -z "$file" ]]; then
+    file="AUTHORS"
+  fi
+fi
+
+if [[ $# = 0 ]]; then
```

Comment:
There seems to be some duplication. There are two sections that have

```sh
authors >> "$somevariable"
exit 0
```

Can they be combined into one? If they share the same variable name, it should be relatively straightforward